src/py42/util.py

```diff
@@ -96,3 +101,29 @@ def to_list(value):
     if not isinstance(value, (list, tuple)):
         return [value]
     return value
+
+
+def parse_timestamp_to_milliseconds_precision(timestamp):
+    if isinstance(timestamp, int) or isinstance(timestamp, float):
+        return convert_timestamp_to_str(timestamp)
+
+    if isinstance(timestamp, datetime):
+        return convert_datetime_to_timestamp_str(timestamp)
+
+    if isinstance(timestamp, str):
+        return convert_datetime_to_timestamp_str(
+            datetime.strptime(timestamp, DATE_STR_FORMAT)
+        )
+
+
+def parse_timestamp_to_microseconds_precision(timestamp):
+    if isinstance(timestamp, int) or isinstance(timestamp, float):
+        return datetime.utcfromtimestamp(timestamp).strftime(MICROSECOND_FORMAT)
+
+    if isinstance(timestamp, datetime):
```

Comment:
These should use `elif`

Reply:
When we have `return` statements does having elif matter?

Reply:
It doesn't matter as much, but if the implementation ever chanages, we may forgot to switch these to use `elif` instead of `if`.. Also, for readability, it improves.

So, 1.) is more maintainable

and 2.). is more readable.

But, it is not that big of a deal, which I why I approved already.

Reply:
Changed.